CUJExecutor/megaTest

```diff
@@ -1,7 +1,25 @@
 #!/usr/bin/env bash
 
-##Loops through all provided compilation flags 10 times and runs manipulateProfilesAndTest
-##Appends the SHORT_SUMMARY from each run to the AGGREGATED_DATA_FILE for the specified INPUT_FILE
+##
+# Loops through all provided compilation flags 5 times, each time recompiling aGMM as specified and measuring the desired portion of the CUJ specified by INPUT_FILE
+# Appends the SHORT_SUMMARY from each run to the AGGREGATED_DATA_FILE for the specified INPUT_FILE
+# Stores the following files at /mega_data/<INPUT_FILE_NAME>/<DATE>/:
+# - (all_data.txt) a log containing formatted data from each measurement (acquired via a call to manipulateProfilesAndTest) 
+# - (summary.txt) a summary containing average action durations for each compilation flag (over all iterations)
+#
+# @param INPUT_FILE (Formatted with the following on consecutive lines: PRE-CUJ, POST_CUJ, NUM_ITERS, and optionally RECORD (a line with an "r" to indicate that you'd like to record). See CUJS/ for an example file.
+# @param COMPILATION_FLAGS (all arguments after INPUT_FILE, e - empty profile, p - compile actions in Pre-CUJ, c - compile entire CUJ, a - compile all code)
+
+# Example: ./megaTest CUJS/takeout.txt e p c a 
+# Result: Will loop through compiling on an empty profile, one containing the pre-CUJ, one containing the entire CUJ and one containing all of aGMM, measuring the desired CUJ each time (as specified by the takeout file)
+#
+# Example: ./megaTest CUJS/takeout.txt a e
+# Result: Will alternate between compiling copiling all of aGMM and compiling nothing, measuring the desired CUJ each time (as specified by the takeout file)
```

Comment:
Typo: `compiling copiling`.

What does `alternate between compiling` mean?

Reply:
compiles all of aGMM, measures, compiles nothing, measures, compiles all of aGMM, measures, compiles nothing, measures, etc.